src/Core/Content/Media/Infrastructure/Url/FallbackUrlGenerator.php

```diff
@@ -0,0 +1,47 @@
+<?php declare(strict_types=1);
+
+namespace Frosh\BunnycdnMediaStorage\Core\Content\Media\Infrastructure\Url;
+
+use Frosh\BunnycdnMediaStorage\Service\CdnHealthCheckService;
+use Shopware\Core\Content\Media\Infrastructure\Url\UrlGeneratorInterface;
+use Shopware\Core\Framework\Context;
+
+class FallbackUrlGenerator implements UrlGeneratorInterface
+{
+    private UrlGeneratorInterface $decorated;
```

Comment:
could you please use class property declaration instead?
The rtrim of localPublicUrl can be moved to the generate method?